frontend/src/metabase/components/MetadataInfo/MetadataInfo.module.css

```diff
@@ -0,0 +1,11 @@
+.Description {
+  white-space: pre-line;
+  max-height: 200px;
+  overflow: auto;
+  margin-bottom: 0.5em;
+
+  .EmptyDescription {
```

Comment:
:x: This should not be nested.
Or we need this to be `&.EmptyDescription`.
But if nesting is not necessary - let's avoid it for the sake of shorter CSS.